js/ui/gallery.js

```diff
@@ -337,6 +338,20 @@ const Gallery = CollectionWidget.inherit({
         this._loadNextPageIfNeeded();
     },
 
+    _createItemByTemplate(itemTemplate, renderArgs) {
+        return itemTemplate.render({
+            model: renderArgs.itemData,
+            container: renderArgs.container,
+            index: renderArgs.index,
+            onRendered: () => {
+                if(!this._resizeCalled) {
```

Comment:
Hm, can we just add an empty `onItemTemplateRendered` handler on collection level and override if here to reduce the copypaste?